nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -125,10 +125,12 @@
    You will need to reboot after enabling this driver to prevent a clash with
    other kernel modules.
   </para>
+  <note>
   <para>
-   Note: for recent AMD GPUs you most likely want to keep either the defaults
+   For recent AMD GPUs you most likely want to keep either the defaults
    or <literal>"amdgpu"</literal> (both free).
   </para>
+  </note>
```

Comment:
Accidentally included this unrelated change

Reply:
I actually did this on purpose: <note> is used inconsistently in this section of the manual. Is it ok to include it in this change?

Reply:
Ah yeah sounds good, but a separate commit for this would be nice

Reply:
Ok, done.